crates/ln-dlc-node/src/node/invoice.rs

```diff
@@ -182,8 +181,13 @@ where
             .iter()
             .any(|channel| channel.counterparty.node_id == trader_id);
         if is_for_jit_channel {
-            let fake_scid = FakeScid::new(intercept_scid);
-            let channel = Channel::new_jit_channel(trader_id, fake_scid);
+            let channel = Channel::new(
+                user_channel_id,
+                0,
+                0,
+                trader_id,
+                liquidity_request.liquidity_option_id,
+            );
```

Comment:
:wrench: I highly recommend to adapt the `new_jit_channel` constructor instead.

Reply:
you convinced me.